datafusion/core/src/execution/context/mod.rs

```diff
@@ -2210,6 +2212,31 @@ impl<'a> TreeNodeVisitor for BadPlanVisitor<'a> {
     }
 }
 
+pub fn create_table_options_from_cmd(
```

Comment:
This is how we create a table config by combining session options and statement options. I believe there is a misusage case I did not fully block, which is defining "format.csv.x" and "format.json.y" at the same time. I will work on that since the current implementation prevents this type of usage.